integration/combination/test_api_with_cors.py

```diff
@@ -12,10 +12,7 @@
 @skipIf(current_region_does_not_support([REST_API]), "Rest API is not supported in this testing region")
 class TestApiWithCors(BaseTest):
     @parameterized.expand(
-        [
-            "combination/api_with_cors",
-            "combination/api_with_cors_openapi",
-        ]
+        ["combination/api_with_cors", "combination/api_with_cors_openapi", "combination/api_with_cors_and_apikey"]
```

Comment:
nit: can you add back the trailing comma so `black` can format on multiple lines? easier to review what changed

Reply:
Will do!